tests/e2e_tests

```diff
@@ -11,8 +11,9 @@ fi
 
 touch error.log
```

Comment:
unrelated to the MR, just noticed it when trying to understand the script. The following doesn't make much sense to me:

if `error.log` exists, then create+truncate it, otherwise touch it.

I think it could be simplified into:
```suggestion
touch error.log
echo '' > error.log
```

or if you do prefer the shorthand version (personally I had to look it up):

```suggestion
:> error.log
```

The suggestion can't capture it, didn't think I would prefer GitLab diff but here we are... I meant to capture the whole block with the if statement + the touch

Reply:
agree, done